po/pt_BR.po

```diff
@@ -0,0 +1,453 @@
+"turn if off and on again."
+msgstr ""
+"Se o dispositivo já encontra-se ligado,\n"
+"desligue-o e ligue-o novamente."
+
+#: lib/solaar/ui/tray.py:55
+msgid "No Logitech receiver found"
+msgstr "Nenhum receptor da Logitech encontrado"
+
+#: lib/solaar/ui/tray.py:62
+msgid "Quit"
+msgstr "Sair"
+
+#: lib/solaar/ui/tray.py:274
+msgid "no receiver"
+msgstr "nenhum receptor"
+
+#: lib/solaar/ui/tray.py:293
+msgid "no status"
+msgstr "sem status"
+
+#: lib/solaar/ui/window.py:58
+msgid "The wireless link between this device and its receiver is encrypted."
+msgstr "O link sem fio entre o dispositivo e seu receptor é criptografado."
+
+#: lib/solaar/ui/window.py:59
+msgid ""
+"The wireless link between this device and its receiver is not encrypted.\n"
+"\n"
+"For pointing devices (mice, trackballs, trackpads), this is a minor security "
+"issue.\n"
+"\n"
+"It is, however, a major security issue for text-input devices (keyboards, "
+"numpads),\n"
+"because typed text can be sniffed inconspicuously by 3rd parties within "
+"range."
+msgstr ""
+"O link sem fio entre o dispositivo e seu receptor é criptografado.\n"
+"\n"
+"Para dispositivos de apontamento (mouses, trackballs, trackpads), isto não representa um grande "
+"problema de segurança.\n"
+"\n"
+"No entando, é um grande problema de segurança para dipositivos de entrada de texto (teclados, "
+"numpads) pois,\n"
+"o texto digitado pode ser capturado, de forma indetectável, por terceiros que estejam dentro do "
+"alcance."
+
+#: lib/solaar/ui/window.py:67 lib/solaar/ui/window.py:71
+msgid "No device paired"
+msgstr "Nenhum dispositivo emparelhado"
+
+#: lib/solaar/ui/window.py:67 lib/solaar/ui/window.py:68
+#, python-format
+msgid "Up to %d devices can be paired to this receiver"
+msgstr "Até %d dispositivos podem ser emparelhados com este receptor"
+
+#: lib/solaar/ui/window.py:68
+msgid "paired devices"
+msgstr "dispositivos emparelhados"
+
+#: lib/solaar/ui/window.py:72
+msgid "Only one device can be paired to this receiver"
+msgstr "Apenas um dispositivo pode ser emparelhado com este receptor"
+
+#: lib/solaar/ui/window.py:110
+msgid "Scanning"
+msgstr "Procurando"
+
+#: lib/solaar/ui/window.py:146
+msgid "Wireless Link"
+msgstr "Link sem fio"
+
+#: lib/solaar/ui/window.py:179
+msgid "Show Technical Details"
+msgstr "Mostrar detalhes técnicos"
+
+#: lib/solaar/ui/window.py:192
+msgid "Pair new device"
+msgstr "Emparelhar novo dispositivo"
+
+#: lib/solaar/ui/window.py:211
+msgid "Select a device"
+msgstr "Selecione um dispositivo"
+
+#: lib/solaar/ui/window.py:508
+msgid "Path"
+msgstr "Caminho"
+
+#: lib/solaar/ui/window.py:510
+msgid "USB id"
+msgstr "id USB"
+
+#: lib/solaar/ui/window.py:513 lib/solaar/ui/window.py:515
+#: lib/solaar/ui/window.py:527 lib/solaar/ui/window.py:529
+msgid "Serial"
+msgstr "Número de série"
+
+#: lib/solaar/ui/window.py:519
+msgid "Index"
+msgstr "Índice"
+
+#: lib/solaar/ui/window.py:520
+msgid "Wireless PID"
+msgstr "PID sem fio"
```

Comment:
Not sure but here I prefer the technical jargon "PID Wireless" because "PID sem fio"  does not bind it with the "wireless link" . However, this is good translation! Comments welcome